docs/ref/models/querysets.txt

```diff
@@ -3567,14 +3567,11 @@ elsewhere.
 
 .. class:: Q
 
-A ``Q()`` object, like an :class:`~django.db.models.F` object, encapsulates a
-SQL expression in a Python object that can be used in database-related
-operations.
-
-In general, ``Q() objects`` make it possible to define and reuse conditions.
-This permits the :ref:`construction of complex database queries
-<complex-lookups-with-q>` using ``|`` (``OR``) and ``&`` (``AND``) operators;
-in particular, it is not otherwise possible to use ``OR`` in ``QuerySets``.
+A ``Q()`` object represents a SQL condition that can be used in
```

Comment:
While you're at this,

[`a` -> `an`](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#commonly-used-terms)

Reply:
Ah right, thanks for pointing me to the guide. Searching Django source and docs, it seems we have some "ess-que-ell" usage, so another PR coming up to fix those.